R4ML/R/sysml.bridge.R

```diff
@@ -82,10 +82,10 @@ sysml.MatrixCharacteristics <- setRefClass("sysml.MatrixCharacteristics",
 sysml.MLContext <- setRefClass("sysml.MLContext",
   fields = list(env="environment"),
   methods = list(
-    initialize = function(sparkContext = sysmlSparkContext) {
+    initialize = function(sparkContext = r4ml.env$sysmlSparkContext) {
       env <<- new.env()
       if (missing(sparkContext)) {
-        sparkContext = get("sysmlSparkContext", .GlobalEnv)
+        sparkContext <- r4ml.env$sysmlSparkContext
```

Comment:
Does this still matter since IOP is gone? If so do you have any ideas about a name for a new environment? Or can we use `r4ml.env` for now and in the future use a new environment just for stuff like this?